upcloud/storage.go

```diff
@@ -54,6 +57,9 @@ type Storage struct {
 	Type       string `xml:"type"`
 	UUID       string `xml:"uuid"`
 	Zone       string `xml:"zone"`
+	// Only for type "backup":
+	Origin  string    `xml:"origin"`
+	Created time.Time `xml:"created"`
```

Comment:
The marshalling for these fields should probably be tested; IIRC there are tests like these that can be used as examples?

Reply:
I just realized that the integration tests work on regular customer accounts, too, if `UPCLOUD_GO_SDK_TEST_USER` and `UPCLOUD_GO_SDK_TEST_PASSWORD` are provided. :tada: From the Travis error message I assumed that some special setup would be required.

Let me see if I can provide a useful test...

Reply:
Be *very aware* that the integration test suite may delete all your resources if you're not careful, I suggest using a throwaway account. See https://github.com/UpCloudLtd/upcloud-go-api#testing.

Reply:
Thanks for the pointer. I haven't set `UPCLOUD_GO_SDK_TEST_DELETE_RESOURCES` so `teardown()` won't be called. I'm also doing this on a non-production account. :slightly_smiling_face: